d2l-hm-constants-behavior.html

```diff
@@ -188,6 +189,11 @@
 					unsubscribe: 'unsubscribe',
 					update: 'update'
 				}
+			},
+			notificationScheduler: {
```

Comment:
should this be a child of `notifications`?

Reply:
It's a separate service working alongside, but separate from notifications.

Reply:
https://github.com/Brightspace/notification-scheduler

Reply:
Should it maybe be named digest-notifications?  or maybe even digest.notifications.api.brightspace.com to indicate it's a subpart of the wider notifications thing? @jopnick might have some ideas?

Reply:
That would require creating a new cert. If we want the <tenantId>.digest.notifications.api.brightspace.com that would be another wildcard cert. I'm not sure how difficult/how much $ this would be. @omsmith @jopnick?

Reply:
Probably not worth doing it then, i wasn't thinking about the cert implications